analysis.ml

```diff
@@ -180,13 +180,16 @@ end
 
 exception DuplicateFormalParameter
 
-let as_var_set (formals : formal_parameter list) =
+let as_var_list (formals : formal_parameter list) =
```

Comment:
`as_var_list` has the advantage of maintaining the order of `formals` which is lost as soon as conversion to `Set` is performed. If we intend to change the type of `analysis_input` to contain a `List` of `formals` rather than a `Set` of `formals`, then `as_var_list` makes more sense.